### DISCUSSION TITLE
Modify speed and intensity after image loading

### DISCUSSION THREAD

**Discussion by linupa** [arkypita/LaserGRBL#2128] [Ideas]
Hi,
We can change the laser speed and intensity only during file opening (or reopening).
Moreover we cannot confirm what we chose after image loading.
So, I think it would be helpful to make a small dialog to show the current laser speed and intensity and modify them.
It would be the same to "Target Image" dialog during file open, but it would be a modal-less or displayed under g-code window.

I am not sure if someone is already working on that, but I think it would be really useful.

**Comment by Tribo-2dot0** [comment]
The speed and power is shown in the top right corner and you can change them here during the process:
![Captura de tela 2023-09-07 010535](https://github.com/arkypita/LaserGRBL/assets/83982525/47cbbec0-51cf-4f23-9022-d014efd68810)
